guidelines/goals.md

```diff
@@ -40,7 +44,12 @@ below.
  4. A specification for resolving translations at runtime, including
     interpolated data types and runtime errors.
 
- 5. A conformance test suite for parsing and formatting messages.
+ 5. A conformance test suite for parsing and formatting messages sufficient to
+    ensure implementations can validate conformance to the specification(s) provided.
+
+ 6. Ensure that there are at least two interoperable independent implementations
+    compliant with the conformance test suite in order to demonstrate that the 
+    specification(s) are pratical and meet requirements.
```

Comment:
This phrasing doesn't quite match the other deliverables; "ensure" doesn't present a concrete thing that we're delivering. Maybe something like this?

```suggestion
 6. A determination that there are at least two interoperable independent implementations
    compliant with the conformance test suite in order to demonstrate that the 
    specification(s) are practical and meet requirements.
```

Reply:
Done.